docs/src/substructure.md

```diff
@@ -0,0 +1,129 @@
+# Jet Substructure
+
+## Structures
```

Comment:
Don't just repeat the docstrings! Write a description of how to use these functions and refer to the API documentation by reference (e.g. `[MassDropTagger](@ref)`).

You can refer to the nice examples you wrote as well.